uber/config.py

```diff
@@ -718,7 +718,7 @@ def GETTABLE_SITE_PAGES(self):
                 of key/value pairs for each page inside that section.
                 Example: 
                     pages['registration'] = [
-                        {'name': 'Arbitrary Charge Form', 'path': '/registration/arbitrary_charge_form'},
+                        {'name': 'Arbitrary Charge Form', 'path': '/merch_admin/arbitrary_charge_form'},
```

Comment:
We should make sure that reg admins have access to the arbitrary charge form after this gets merged.